cmd/operator/main.go

```diff
@@ -95,40 +114,48 @@ func main() {
 
 	withWebhook := true
 
-	// TODO(mythi): add a StringVar flag to select which controllers to enable
-	for _, device := range []string{"dsa", "fpga", "gpu", "qat", "sgx"} {
+	for device, enabled := range devices {
+		if !enabled {
```

Comment:
Oops, this condition will never be true I guess, which makes it useless to use type flagMap map[string]bool.

Reply:
Yes, I noticed the same. The string array fits here better. OK, I will change it back.